src/core_plugins/kibana/public/home/components/tutorial/saved_objects_installer.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiSteps,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiButton,
+  EuiSpacer,
+  EuiCallOut,
+} from '@elastic/eui';
+
+const INCOMPLETE = 'incomplete';
+const COMPLETE = 'complete';
+
+export class SavedObjectsInstaller extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isInstalling: false,
+      installStatus: INCOMPLETE,
+    };
+  }
+
+  installSavedObjects = async () => {
+    this.setState({
+      isInstalling: true,
+    });
+
+    const resp = await this.props.bulkCreate(this.props.savedObjects, { overwrite: true });
+    const errors = resp.savedObjects.filter(savedObjectCreateResult => {
+      return savedObjectCreateResult.hasOwnProperty('error');
+    });
+
+    let statusMsg = `${this.props.savedObjects.length} saved objects successfully added`;
+    if (errors.length > 0) {
+      statusMsg = `Unable to load kibana saved objects, Error: ${errors[0]}`;
+    }
+
+    this.setState({
+      isInstalling: false,
+      installStatusMsg: statusMsg,
+      installStatus: errors.length === 0 ? COMPLETE : INCOMPLETE,
```

Comment:
Instead of having two different checks for error (`errors.length === 0` and `errors.length > 0`) maybe just have one `const hasError = errors.length > 0`. 

This way they won't somehow come out of sync in a future change.